.github/workflows/terraform-python.yaml

```diff
@@ -43,11 +43,6 @@ permissions:
 jobs:
   terraform:
     runs-on: ubuntu-latest
-
-    defaults:
-      run:
-        working-directory: ./tf/main
```

Comment:
this will break existing projects, so we need to backfill existing projects with a version tag?